### PR TITLE
#167572351  Validates User Sign-in Input

### PR DESCRIPTION
**Pull request by OdunayoOkebunmi**
### What does this PR do?
validates user signin details with express-validator
writes tests for validation
document using swagger.json
#### Description of Task to be completed?
GIVEN user supplies invalid parameters for signin
WHEN user hits the signin button
THEN user should receive a descriptive error message telling the user why such details are not acceptable.
#### How should this be manually tested?
1. Fetch this branch on the terminal using 
`git fetch origin ft-validate-signin-167572351`
2. Install dependencies using `npm install`
3. Start the server with `npm run server`
4. On Postman, sign up by making a `POST` request to `/api/v1/auth/signin`

#### What are the relevant pivotal tracker stories?
[#167572351](https://www.pivotaltracker.com/story/show/167572351)

#### Screenshots (if appropriate)

Screenshot 2019-07-29 at 18 34 08
Screenshot 2019-07-29 at 18 34 03
Screenshot 2019-07-29 at 18 33 31

### CODE REVIEW

File: __tests__/inputValidation.test.js [hunk 1]
```diff
@@ -76,3 +78,18 @@ describe('User tests', () => {
       });
   });
 });
+
+describe('User signup validation test', () => {
```
Comment (mazma1) [inline __tests__/inputValidation.test.js]: Signup or signin?

File: __tests__/inputValidation.test.js [hunk 2]
```diff
@@ -76,3 +78,18 @@ describe('User tests', () => {
...
+  it('should return status 400 if the email input is invalid', (done) => {
```
Comment (mazma1) [inline __tests__/inputValidation.test.js:83]: No test for invalid password input?

**Review by mazma1 — CHANGES_REQUESTED** [review]

File: __tests__/inputValidation.test.js (hunk 1 above)
Comment (OdunayoOkebunmi) [reply __tests__/inputValidation.test.js]: sign-in

File: __tests__/inputValidation.test.js (hunk 2 above)
Comment (OdunayoOkebunmi) [reply __tests__/inputValidation.test.js:83]: I'll add that now

**Pull request merged**